### DISCUSSION TITLE
有没有大佬给个和vue后台一样的多标签页模板

### DISCUSSION THREAD

**Discussion by komorebi-cqd** [ant-design/ant-design#50487] [General]
![微信图片_20240819174220](https://github.com/user-attachments/assets/b0d676dc-6202-4404-af68-2299f38d9aa5)
有没有大佬给个和vue后台一样的多标签页模板，自己写全是问题....

**Comment by afc163** [comment]
试试 https://alitajs.com/docs/guides/tabs#getcustomtabs

**Reply by komorebi-cqd** [reply]
试了，太多bug了。
开启很多标签页不会自己滚动到对应的标签，右侧直接看不见了
![bug1](https://github.com/user-attachments/assets/1473b9a6-9664-4710-80d6-2afb259d4114)

切换主题色不会变化，
![bug2](https://github.com/user-attachments/assets/f51d92c4-b07b-45d6-8c14-cec7492049fb)

点击pro自带的标签跳转，百分百无限重复渲染
![bug3](https://github.com/user-attachments/assets/1d82b54c-1b1d-4322-a47b-716ddd65590f)

首次进入页面不会把padding设为0，点别的路由才会加上这个样式
![bug4](https://github.com/user-attachments/assets/f11d09bb-79a2-4d6f-9daa-4e5624be040d)
